cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -585,7 +585,7 @@ pub mod pallet {
 			// TODO: This is more than zero, but will need benchmarking to figure out what.
 			let mut total_weight = Weight::zero();
 
-			// NOTE: the inherent data is expected to be unique, even if this block is built
+			// NOTE: the inherent data is expected to be unique, even if this block is build
```

Comment:
```suggestion
			// NOTE: the inherent data is expected to be unique, even if this block is built
```

Don't mean to be 'the high school English teacher' here but I believe 'built' was used correctly here :)

Reply:
But the block "is build" and not "was built"